mangum/handlers/api_gateway.py

```diff
@@ -120,8 +121,10 @@ def __call__(self, response: Response) -> dict:
 
         return {
             "statusCode": response["status"],
-            "headers": finalized_headers,
-            "multiValueHeaders": multi_value_headers,
+            "headers": handle_exclude_headers(finalized_headers, self.config),
+            "multiValueHeaders": handle_exclude_headers(
+                multi_value_headers, self.config
+            ),
```

Comment:
There might be a smarter way to manipulate the headers, but I'm not much familiar with the other services like ALB so maybe instead of adding a specific `handle_exclude_headers ` it could be added to one of the methods building the response.

Reply:
I prefer this to the smarter way, easy to understand. :)